src/apify_client/_utils.py

```diff
@@ -57,6 +57,13 @@ def _pluck_data(parsed_response: Any) -> Dict:
     raise ValueError('The "data" property is missing in the response.')
 
 
+def _pluck_data_as_list(parsed_response: Any) -> List:
+    if isinstance(parsed_response, dict) and 'data' in parsed_response:
+        return cast(List, parsed_response['data'])
+
+    raise ValueError('The "data" property is missing in the response.')
+
+
```

Comment:
are you using this?

Reply:
Shouldn't things like this be done where the request is made? (ResourceClient, I think)
Seems it could be done universally for all requests.

Reply:
@fnesveda

Reply:
Or did I misunderstand you and your question is why the function is in _utils.py? If that's the case, then I am not completely sure, but it shouldn't matter.

Reply:
Oh, I remember now. To decrease the probability of merging conflicts, I stole the entire file from one of your branches.

Reply:
The question was why this function is here because it seems not be used in this branch. Now I understand you grabbed a file from different branch. Still, please remove this function if you don't use it in this code. You have no guarantee this file in that other branch changes and they you'd accidentally check-in dead code. Thx